.github/workflows/check_release_tag.py

```diff
@@ -1,16 +1,32 @@
 # -*- coding: utf-8 -*-
 """Check that the GitHub release tag matches the package version."""
 import argparse
-import json
+import ast
+from pathlib import Path
+
+
+def get_version_from_module(content: str) -> str:
+    """Get the __version__ value from a module."""
```

Comment:
I take it that you employ this (convoluted) way of reading the version number because you cannot import it normally? Instead of this approach, is there no simple way to make the package importable so we can just do
```
import aiida
version = aiida.__version__
```

At the very least, I think it would be useful to have a comment here explaining why this approach is necessary and the import way doesn't work or is not desirable

Reply:
I wouldn't call this approach particularly convoluted; I would say having to go through the whole process of installing/importing the package is convoluted, just to read the version from a file 🤷

Reply:
Feel free to create a `suggestion` for what the docstring should be, but I'm not adding one, because I don't feel it needs it, since this IMO is the canonical way to retrieve the `__version__` attribute (it is how both setuptools and flit do it)

Reply:
If generating an abstract syntax tree, parsing it element by element just to find the one string that defines the version number is not convoluted, then I don't know what is. I can see the point that having to install it is likewise a lot of work just to get the version, but at least it is very understandable and readable. One would think that there should be an easier way to do this...  But in the absence of that, guess this is fine to keep

Reply:
I already agreed to keeping this and move forward. Still, the suggestion to add a comment explaining why this is being done doesn't seem like a big or weird ask. Even if this may be the standard, I doubt many developers are aware of this and they will almost certainly wonder why it has to be (or look) so complicated.

Reply:
I would note, that on the next line, it says: `# adapted from setuptools/config.py`, but yeh honestly this is just a very simple check (that wasn't even here a year ago) to make sure the release tag is consistent with the package version